api/api.go

```diff
@@ -17,6 +17,7 @@ type ClientService interface {
 	SyncNotify(flux.InstanceID) error
 	SyncStatus(flux.InstanceID, string) ([]string, error)
 	UpdatePolicies(flux.InstanceID, flux.PolicyUpdates) (job.ID, error)
+	LogEvent(inst flux.InstanceID, e flux.Event) error
```

Comment:
I think this should go into the `DaemonService`, since it's for daemons to call. (It doesn't make much difference, it's just layout, but good to be consistent).